functions/HttpTriggerAPIUsersIdTaskId/__init__.py

```diff
@@ -6,71 +10,124 @@
 #GET API method function
 def get(param1, param2):
     #connects to db
+    logging.debug('Attempting a db conenction')
     cnxn = connect()
-    cursor = cnxn.cursor()
-    logging.info('opened connection')        
-    logging.info('Going to execute select query')
+    cursor = cnxn.cursor() 
     try:
+        logging.debug('opened connection')        
+        logging.debug('Attempting to execute GET task query')
         #Get task title, description and user name by userId and taskId
-        sql_query = ("""SELECT tasks.title, tasks.description, CONCAT (users.firstName, ' ', users.lastName) AS "user" 
-        FROM [dbo].[tasks] JOIN [dbo].[users] 
-        on [dbo].[tasks].userId = [dbo].[users].userId
-        WHERE [dbo].[users].userId = ? AND [dbo].[tasks].taskId = ?""")
+        sql_query = ("""SELECT tasks.title, tasks.description, tasks.createdDate, tasks.dueDate, 

```

Comment:
another shorter alternative =>
"Select * from tasks where userId = ? and taskId = ?"
of course, if we don't want to have the user's name. just a suggestion.

Reply:
Makes sense, and thank you for pointing this out. We could still have a join and select * from tasks, and user name from users. My thinking is along the lines of, if we need to expand the database in the future, we do not have all of the contents of the table exposed through the API, and selecting specific columns will help with that